Ginger/GingerCoreNETUnitTest/RunTestslib/UnitTestRepositoryItemFactory.cs

```diff
@@ -177,5 +177,10 @@ public bool IsSharedRepositoryItem(RepositoryItemBase item)
         {
             return SharedRepositoryOperations.IsSharedRepositoryItem(item);
         }
+
+        public void DispatcherRun()
```

Comment:
it does not need to throw exception as well

Reply:
I have removed exception code.